tools/convert.sh

```diff
@@ -280,18 +280,16 @@ if ! test -f "${mdOutFile}"; then
   warning 'Unable to merge markdown files, no content found!'
   exit 1
 fi
-mdContent=$(cat "${mdOutFile}")
-if test -n "${ReplaceFile}"; then
-  if ! test -f "${replaceFilePath}"; then
-    error '' "Unable to find replace file ${replaceFilePath}" 1
-  fi
-  info 'Perform replace in markdown'
-  tab_values=$(jq -r 'to_entries[] | [.key, .value] | @tsv' "${replaceFilePath}")
-  printf '%s\n' "${tab_values}" | while IFS="$(printf '\t')" read -r key value; do
-    printf '%s\n' "${mdContent}" | sed -e "s/${key}/${value}/g" > "${mdOutFile}"
-  done
-  mdContent="$(cat "${mdOutFile}")"
+
```

Comment:
We need to enclose line 284 to 292 with

```sh
if test -n "${ReplaceFile}"; then
fi
```

This is there because ReplaceFile is optional, we should not error if it is not specified.

Reply:
Fixed